2022-01-20 17:20:47,509 INFO    MainThread:30424 [wandb_setup.py:_flush():71] setting env: {}
2022-01-20 17:20:47,509 INFO    MainThread:30424 [wandb_setup.py:_flush():71] setting login settings: {}
2022-01-20 17:20:47,509 INFO    MainThread:30424 [wandb_init.py:_log_setup():371] Logging user logs to C:\Users\rb01243\OneDrive - University of Surrey\Documents\GitHub\dental_disease\notebooks\wandb\run-20220120_172047-1u4kzi04\logs\debug.log
2022-01-20 17:20:47,509 INFO    MainThread:30424 [wandb_init.py:_log_setup():372] Logging internal logs to C:\Users\rb01243\OneDrive - University of Surrey\Documents\GitHub\dental_disease\notebooks\wandb\run-20220120_172047-1u4kzi04\logs\debug-internal.log
2022-01-20 17:20:47,509 INFO    MainThread:30424 [wandb_init.py:_jupyter_setup():321] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x0000020B0AC8AD00>
2022-01-20 17:20:47,510 INFO    MainThread:30424 [wandb_init.py:init():404] calling init triggers
2022-01-20 17:20:47,510 INFO    MainThread:30424 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {'bcc_epoch': -1, 'qtfilter_epoch': -1, 'qt_thres_mode': '', 'qt_thres': 0.0, 'hybrid_entropy_thres': 0.0, 'hybrid_conf_thres': 0.0, 'weights': 'yolov5s.pt', 'cfg': '../src/models/yolov5s.yaml', 'data': '../data/iid-tvt.yaml', 'hyp': {'lr0': 0.01, 'lrf': 0.2, 'momentum': 0.937, 'weight_decay': 0.0005, 'warmup_epochs': 3.0, 'warmup_momentum': 0.8, 'warmup_bias_lr': 0.1, 'box': 0.05, 'obj': 1.0, 'cls': 0.5, 'cls_pw': 1.0, 'obj_pw': 1.0, 'iou_t': 0.2, 'anchor_t': 4.0, 'fl_gamma': 0, 'hsv_h': 0.015, 'hsv_s': 0.7, 'hsv_v': 0.4, 'degrees': 0.0, 'translate': 0.1, 'scale': 0.5, 'shear': 0.0, 'perspective': 0.0, 'flipud': 0.0, 'fliplr': 0.5, 'mosaic': 1.0, 'mixup': 0.0, 'copy_paste': 0.0}, 'epochs': 18, 'batch_size': 2, 'imgsz': 640, 'rect': False, 'resume': False, 'nosave': False, 'noval': False, 'noautoanchor': False, 'evolve': None, 'bucket': '', 'cache': None, 'image_weights': False, 'device': '', 'multi_scale': False, 'single_cls': False, 'adam': False, 'sync_bn': False, 'workers': 8, 'project': 'runs/train', 'entity': None, 'name': 'test2099', 'exist_ok': False, 'quad': False, 'linear_lr': False, 'label_smoothing': 0.0, 'upload_dataset': False, 'bbox_interval': -1, 'save_period': -1, 'artifact_alias': 'latest', 'local_rank': -1, 'freeze': 0, 'patience': 1100, 'save_dir': 'runs\\train\\test209914'}
2022-01-20 17:20:47,510 INFO    MainThread:30424 [wandb_init.py:init():460] starting backend
2022-01-20 17:20:47,510 INFO    MainThread:30424 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=spawn, using: spawn
2022-01-20 17:20:47,514 INFO    MainThread:30424 [backend.py:ensure_launched():216] starting backend process...
2022-01-20 17:20:47,583 INFO    MainThread:30424 [backend.py:ensure_launched():221] started backend process with pid: 25924
2022-01-20 17:20:47,584 INFO    MainThread:30424 [wandb_init.py:init():469] backend started and connected
2022-01-20 17:20:47,593 INFO    MainThread:30424 [wandb_run.py:_label_probe_notebook():911] probe notebook
2022-01-20 17:20:47,595 INFO    MainThread:30424 [wandb_run.py:_label_probe_notebook():921] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2022-01-20 17:20:47,595 INFO    MainThread:30424 [wandb_init.py:init():533] updated telemetry
2022-01-20 17:20:47,595 INFO    MainThread:30424 [wandb_init.py:init():563] communicating current version
2022-01-20 17:20:48,971 INFO    MainThread:30424 [wandb_init.py:init():568] got version response 
2022-01-20 17:20:48,971 INFO    MainThread:30424 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-01-20 17:20:49,329 INFO    MainThread:30424 [wandb_init.py:init():606] starting run threads in backend
2022-01-20 17:20:54,344 INFO    MainThread:30424 [wandb_run.py:_console_start():1810] atexit reg
2022-01-20 17:20:54,344 INFO    MainThread:30424 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.WRAP
2022-01-20 17:20:54,344 INFO    MainThread:30424 [wandb_run.py:_redirect():1721] Wrapping output streams.
2022-01-20 17:20:54,346 INFO    MainThread:30424 [wandb_run.py:_redirect():1745] Redirects installed.
2022-01-20 17:20:54,346 INFO    MainThread:30424 [wandb_init.py:init():633] run started, returning control to user process
2022-01-20 17:20:54,348 INFO    MainThread:30424 [wandb_run.py:_config_callback():956] config_cb None None {'data_dict': {'path': '../data/datasets/iid-tvt', 'train': '..\\data\\datasets\\iid-tvt\\images\\train', 'val': '..\\data\\datasets\\iid-tvt\\images\\val', 'test': '..\\data\\datasets\\iid-tvt\\images\\test', 'nc': 2, 'names': ['bone-loss', 'dental-caries']}}
2022-01-20 17:21:17,605 INFO    MainThread:30424 [wandb_init.py:_pause_backend():284] pausing backend
2022-01-20 17:21:17,605 INFO    MainThread:30424 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-01-20 17:21:49,913 INFO    MainThread:30424 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 0
2022-01-20 17:21:49,914 INFO    MainThread:30424 [wandb_run.py:_restore():1752] restore
2022-01-20 17:21:50,823 INFO    MainThread:30424 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 789
  total_bytes: 789
}

2022-01-20 17:21:51,020 INFO    MainThread:30424 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 789
  total_bytes: 789
}

2022-01-20 17:21:51,162 INFO    MainThread:30424 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 789
  total_bytes: 46294
}

2022-01-20 17:21:51,275 INFO    MainThread:30424 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 789
  total_bytes: 46294
}

2022-01-20 17:21:51,384 INFO    MainThread:30424 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 29503
  total_bytes: 46294
}

2022-01-20 17:21:51,492 INFO    MainThread:30424 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 46294
  total_bytes: 46294
}

2022-01-20 17:21:51,599 INFO    MainThread:30424 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 46294
  total_bytes: 46294
}

2022-01-20 17:21:51,707 INFO    MainThread:30424 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 46294
  total_bytes: 46294
}

2022-01-20 17:21:51,818 INFO    MainThread:30424 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 46294
  total_bytes: 46294
}

2022-01-20 17:21:51,927 INFO    MainThread:30424 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 46294
  total_bytes: 46294
}

